src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -47,29 +42,38 @@ define(function (require) {
      * @return {Function} Creates the map
      */
     TileMap.prototype.draw = function () {
+
       var self = this;
       var $elem = $(this.chartEl);
       var div;
-      var worldBounds = L.latLngBounds([-90, -180], [90, 180]);
-
+      var worldBounds = L.latLngBounds([-90, -220], [90, 220]);
 
       // clean up old maps
-      _.invoke(self.maps, 'destroy');
+      _.invoke(self.maps, self.destroy);
       // create a new maps array
       self.maps = [];
 
       return function (selection) {
         selection.each(function (data) {
-          div = $(this);
-          div.addClass('tilemap');
 
-          if (self._attr.lastZoom) {
-            mapZoom = self._attr.lastZoom;
+          var mapCenter = [15, 5];
+          var mapZoom = 2;
+
+          if (self._attr.mapZoom) {
+            mapZoom = self._attr.mapZoom;
+          } else {
+            self._attr.mapZoom = mapZoom;
           }
-          if (self._attr.lastCenter) {
-            mapCenter = self._attr.lastCenter;
+          if (self._attr.mapCenter) {
```

Comment:
Is this a typo? The same thing happens regardless of the conditional. Also, the map center point and zoom are lost completely in this PR - is this why?